ytfzf

```diff
@@ -949,6 +955,8 @@ get_search_query () {
 user_selection () {
 	#remove subscription separators
 	videos_data_clean=$(printf "%s" "$videos_data" | sed "/.*$tab_space$/d")
+    
+    [ "$enable_loop" -eq 1 ] && loop_binds=",enter:execute-silent(sh $0 -U \"selected_data_from_file \"{+f})"
```

Comment:
add some variable maybe `fzf_select_bindings="${fzf_select_bindings-enter,double-click}"` instead of hard coding enter here.